visual-testing-app/src/App.js

```diff
@@ -22,41 +32,43 @@ const allComponents = componentsContext.keys().reduce((components, file) => {
 class App extends React.Component {
   render() {
     return (
-      <Router>
-        <Switch>
-          <Route
-            path="/"
-            exact
-            component={() => (
-              <div>
-                <h1>Visual Testing App</h1>
-                <ul>
-                  {Object.values(allComponents).map(Component => (
-                    <li key={Component.routePath}>
-                      <a href={Component.routePath}>{Component.routePath}</a>
-                    </li>
-                  ))}
-                </ul>
-              </div>
-            )}
-          />
-          {Object.values(allComponents).map(Component => (
+      <ThemeProvider theme={customProperties}>
```

Comment:
Nitpick: maybe instead of spreading the custom properties, should we create and export a "default theme". Then creating new themes would be easier, as you can start by copying the "default theme".

Anyway, just a side thought. It might make more sense to look into this once we've implemented theming to all components.

Reply:
Yeah a default theme makes a lot of sense.